src/test/functional/code/bucle.ts

```diff
@@ -0,0 +1,40 @@
+export const codeInput = `// SWMDE v0.1
+// Autor: Iván Castilla Rodríguez
+// Utilidad: Programa de testeo de SWMDE
+// Comentarios: El programa presupone q en la posición 50 (R2) de memoria tienes un vector de
```

Comment:
Doesn't have to be changed now as it is low priority, but these comments will have to be translated to English in the future.

Reply:
see #80